xivo_dao/resources/switchboard/tests/tests_dao.py

```diff
@@ -61,6 +61,20 @@ def test_search_multi_tenant(self):
         tenants = [tenant.uuid]
         self.assert_search_returns_result(expected, tenant_uuids=tenants)
 
+    def test_search_by_extension(self):
+        switchboard = self.add_switchboard()
+        incall = self.add_incall()
+        self.add_extension(type='incall',
+                           typeval=str(incall.id),
+                           exten="1000")
+        self.add_dialaction(category='incall',
+                            categoryval=str(incall.id),
+                            action='switchboard',
+                            actionarg1=switchboard.uuid)
```

Comment:
Follow the code file style.
This repo is not really standard (i.e. black is not applied by default), but we should not use two styles in the same file
```suggestion
        self.add_extension(
            type='incall',
            typeval=str(incall.id),
            exten="1000",
        )
        self.add_dialaction(
            category='incall',
            categoryval=str(incall.id),
            action='switchboard',
            actionarg1=switchboard.uuid,
        )
```